RELEASE_NOTES.rst

```diff
@@ -29,6 +29,7 @@ ARTIQ-5
 * ``aqctl_corelog`` now filters log messages below the ``WARNING`` level by default.
   This behavior can be changed using the ``-v`` and ``-q`` options like the other
   programs.
+* On Kasli the firmware now starts with a unique default MAC address from EEPROM.
```

Comment:
Only a MAC address isn't programmed in the flash config, right?

Reply:
Right

Reply:
Then that should be pointed out. Your sentence could mean that the EEPROM now overrides everything.